src/workerd/io/actor-sqlite.c++

```diff
@@ -473,9 +474,33 @@ kj::Maybe<kj::Promise<void>> ActorSqlite::put(Key key, Value value, WriteOptions
 
 kj::Maybe<kj::Promise<void>> ActorSqlite::put(kj::Array<KeyValuePair> pairs, WriteOptions options) {
   requireNotBroken();
-
-  for (auto& pair: pairs) {
-    kv.put(pair.key, pair.value);
+  if (util::Autogate::isEnabled(util::AutogateKey::SQL_KV_PUT_MULTIPLE_TRANSACTION)) {
```

Comment:
It's probably worth leaving a cleanup comment in this block that this code is basically stolen from `DurableObjectStorage::transactionSync`.